### PR TITLE
Add createdAt to member state

### PR DESCRIPTION
**Pull request by tibetsprague**
Needed to fix https://github.com/daostack/alchemy/issues/1593

### CODE REVIEW

File: src/member.ts [hunk 1]
```diff
@@ -219,6 +221,7 @@ export class Member implements IStateful<IMemberState> {
         return {
             address: item.address,
             contract: item.contract,
+            createdAt: Number(item.createdAt),
```
Comment (dkent600) [inline src/member.ts:224]: Just curious:  Why not return a Date?

Comment (tibetsprague) [reply src/member.ts:224]: Date is an alias for number in this library, we use numbers for dates right now

**Comment by tibetsprague** [comment]
@jellegerbrandy i do define the type as a date, and just copied what was happening elsewhere. also the spec already catches checks this by comparing states. it caught a bug for me :)

**Comment by tibetsprague** [comment]
actually i am confused about dates. there is a Date type which is an alias of Number. and sometimes we just assign Number(value) to it. but other times we call secondSinceEpochToDate which returns an actual Javascript Date object. How does that map to the number Date type? that is confusing as hell. i dont even get how that works

**Comment by dkent600** [comment]
I'm wondering it this'll all be cleaned up in version arc.js 2.0?

**Comment by jellegerbrandy** [comment]
lots of cleanup thre, but I'm not sure this caught anyone's attention..

**Comment by tibetsprague** [comment]
@jellegerbrandy @dOrgJelli so what shall we do here? Can we merge as is?

**Comment by dOrgJelli** [comment]
It LGTM @tibetsprague, we can make sure to get this change into arc.js 2.0 (cc @namesty)

**Comment by orenyodfat** [comment]
> It LGTM @tibetsprague, we can make sure to get this change into arc.js 2.0 (cc @namesty)

we need that both for current arc.js and for 2.0 @dOrgJelli .
what is stopping us to merge it currently ?

**Pull request merged**